client/src/common/components/Home.tsx

```diff
@@ -3,164 +3,198 @@ import { Link } from 'react-router-dom';
-				<h1 className="text-3xl font-bold md:text-7xl mt-5 text-center md:text-left col-start-1 col-span-3">
-					Senpai Kohai
-				</h1>
-				<img
-					className="md:w-6/6 md:row-span-4 md:col-span-4 mt-5"
-					src="./assets/shared_workspace.svg"
-					alt="shared_workspace"
-				/>
-				<p className="mt-5 font-bold text-base w-auto md:text-2xl text-center md:text-left col-span-3">
-					Find your guide, Support youngs
-				</p>
-				<p className="underline cursor-pointer mt-5 w-auto text-sm md:text-lg text-center md:text-left flex col-span-3 justify-self-center md:justify-self-start">
-					<FaQuestionCircle />
-					&nbsp; What are Senpai and Kohai?
-				</p>
-				<div
-					className="cursor-pointer 
+  // test, added style here just to see the classes work
+  return (
+    <main className="bg-primary_bg_color text-primary_title_color font-sans overflow-x-hidden mt-laptopHeaderHeight tablet_md_max:mt-mobileHeaderHeight py-paddingAroundtheContent ">
+      <section className="flex justify-evenly px-laptopPxAroundtheContent tablet_md_max:px-mobileHeaderHeight mb-20">
+        <div className="flex flex-col justify-between tablet_md_max:items-center">
+          <div className="mt-7 font-bold">
+            <h1 className="lg:text-7xl tablet_l_max:text-5xl tablet_md_max:text-center mobile_m_max:hidden">
+              Senpai Kohai
+            </h1>
+            <p className="mt-5 text-2xl tablet_l_max:text-xl mobile_m_max:hidden">
+              Find your guide, Support youngs
+            </p>
+            {/* ↓ for mobile version */}
+            <h1 className="text-center text-4xl mobile_m:hidden">
+              Senpai Kohai
+            </h1>
+            <p className="mt-5 text-lg text-center mobile_m:hidden">
+              Find your guide, Support youngs
+            </p>
+          </div>
+          <img
+            className="w-80 my-8 md:hidden"
+            src="./assets/shared_workspace.svg"
+            alt="shared_workspace"
+          />
+          <div className="flex mt-5 items-center text-lg mobile_m_max:text-sm">
+            <FaQuestionCircle />
+            <p className="underline cursor-pointer ml-2">
+              What are Senpai and Kohai?
+            </p>
+          </div>
+          <Link
+            to="/signup"
+            className="tablet_md_max:w-full tablet_md_max:flex tablet_md_max:justify-center"
+          >
+            <button
+              className="cursor-pointer 
 								mt-5 
-								border-2 
-								py-2
-								md:py-1.5
-								w-28
-								h-10 
+								py-3.5
+								w-48
 								text-primary_bg_color 
 								bg-primary_title_color 
 								rounded-full 
-								font-black 
+								font-black
 								hover:bg-secondary_bg_color 
-								text-center
-								text-sm
-								md:text-base
-								col-end-2
-								justify-self-center
-								md:justify-self-start
+								md:text-lg
+								tablet_md_max:w-80
+								mobile_m_max:hidden
+								tablet_l_max:p-2.5
```

Comment:
```suggestion
								items-center
                                                                shadow-lg	
```
Added the box-shadow and remove the width for other devices.
We might wanna adjust the button size later.